tests/fixtures/yaml_tests/failing_test_variable_absolute_error_margin.yaml

```diff
@@ -0,0 +1,9 @@
+- name: "Failing test: result out of variable specific absolute error margin"
```

Comment:
I'm not clear here: the absolute error margin is for the whole calculation or just for the variable?

Reply:
The default is for all the variables that are not directly specified.
Imagine you set a 1% relative error margin. 
```yaml
relative_error_margin: .01
```
But you find out that want to have a less strict, say 5%, error margin on some variables.
You can easily modify the previous line to:
```yaml
relative_error_margin: 
- default: .01
- my_variable: .05
```